lib/rails_multitenant/global_context_registry.rb

```diff
@@ -85,6 +85,25 @@ def replace_registry(registry)
       self.globals = registry
     end
 
+    # Run a block of code that disregards scoping during read queries
+    def with_unscoped_queries
+      with_merged_registry(use_unscoped_queries: true) do
```

Comment:
Think we should use a key with an `__` prefix (i.e. `__use_unscoped_queries`) to avoid conflicts with any keys applications may be using?